README.md

```diff
@@ -335,13 +335,47 @@ client.setAnalyticsTag('Navigation search');
 
 ### Personalization
 
+#### Enable personalization tracking
+Enable personalization tracking, personalization token will be included in every stat events as "session ID". <br/>
+
+Set stats session ID if personalization token is generated by your site.
+```js
+client.setStatsSessionId(id);
+```
+
+If session ID is not set, a UUID is generated and stored in AddSearch's cookie. Cookie's name: addsearchPersonalizationToken
+Specify how many days personalization token stay in cookie. By default, it's 180 days.
+```js
+// Defaults - isEnabled: false, expirationDays: 180
+client.enablePersonalizationTracking(isEnabled, expirationDays);
+```
+
+
+#### Allow storing personalization token in cookie
+By default, the value is true. Set it to false when users disallow usage of functional/analytics cookie. <br/>
```

Comment:
`... Set it to false when user does not give consent to usage of functional/analytics cookie...`

Reply:
![Screenshot 2023-05-02 at 11 31 32](https://user-images.githubusercontent.com/58325932/236144070-2c57003c-1be5-4a88-ba02-a596136046ad.png)

Usage of cookies are categorized into 4 common use cases. In Shopify context, we define the usage of cookie is for functional/analytics purposes.

Maybe the description can be improved somehow? It's like a reminder for customers who use js-client so that they  wouldn't call `client.consentAddSearchCookie(false)` when user rejects Marketing cookie.